execute/table/copy.go

```diff
@@ -1,6 +1,8 @@
 package table
 
-import "github.com/influxdata/flux"
+import (
```

Comment:
No problem with this but might as well revert this file since nothing else changed.

Reply:
Ah, I missed this. That was just my formatter plugin. I'll revert the change.